homeassistant/components/influxdb.py

```diff
@@ -77,6 +77,11 @@ def influx_event_listener(event):
             _state = 0
         else:
             _state = state.state
+            if _state and isinstance(_state, str):
```

Comment:
isnumeric() responds flase if the string is a float. For instance my thermostat reports '68.5' which does not pass isnumeric().